aggregator/options.go

```diff
@@ -275,42 +350,35 @@ func (o *options) AggregationCountSuffix() []byte {
 	return o.countSuffix
 }
 
-func (o *options) SetAggregationStdevSuffix(value []byte) Options {
-	opts := *o
-	opts.stdevSuffix = value
-	return &opts
-}
-
-func (o *options) AggregationStdevSuffix() []byte {
-	return o.stdevSuffix
+func (o *options) AggregationSumSuffix() []byte {
+	return o.sumSuffix
 }
 
-func (o *options) SetAggregationMedianSuffix(value []byte) Options {
+func (o *options) SetAggregationSumSqSuffix(value []byte) Options {
 	opts := *o
-	opts.medianSuffix = value
+	opts.sumSqSuffix = value
 	return &opts
 }
 
-func (o *options) AggregationMedianSuffix() []byte {
-	return o.medianSuffix
+func (o *options) AggregationSumSqSuffix() []byte {
+	return o.sumSqSuffix
 }
 
-func (o *options) SetDefaultTimerAggregationTypes(aggTypes policy.AggregationTypes) Options {
+func (o *options) SetAggregationStdevSuffix(value []byte) Options {
 	opts := *o
-	opts.defaultTimerAggregationTypes = aggTypes
-	opts.computeQuantiles()
-	opts.computeDefaultTimerAggregationSuffix()
+	opts.stdevSuffix = value
```

Comment:
When setting each of these suffixes, don't we need to recompute the default suffixes and all the suffixes for all metric types, since it changes the default suffixes and therefore potentially the default type-specific suffixes etc?

If so, you might wanna move all the suffix computing functions into a single one and call it from each of the individual suffix setters.